packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -191,6 +191,42 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
     );
   }
 
+  public getDomainMargins() {
```

Comment:
Are these functions used anywhere in the product code.

Reply:
no, added public handlers for testing of the private functions